src/backend/proxy.ts

```diff
@@ -16,22 +16,13 @@ const restream = (proxyReq: ClientRequest, req: IncomingMessage) => {
     }
 };
 
-export const doProxy = (
-    context: string,
-    targetUrl: string,
-    pathPrefix = '/api'
-): RequestHandler => {
-    return createProxyMiddleware(context, {
+export const doProxy = (context: string, targetUrl: string): RequestHandler => {
+    return createProxyMiddleware({
         changeOrigin: true,
-        logLevel: 'info',
-        logProvider: () => {
-            return stdoutLogger;
-        },
-        onProxyReq: restream,
-        pathRewrite: (path: string) => {
-            const newPath = path.replace(context, '');
-            return `${pathPrefix}${newPath}`;
+        on: {
+            proxyReq: restream,
         },
+        pathRewrite: (path: string) => path.replace(context, ''),
```

Comment:
hva skjedde med path-prefix? Trengs det ikke?
Kan sende med `logger: stdoutLogger` for at loggene skal bli "tolkbare" for Kibana

Reply:
Det fungerte ikke slikt det var over, men tanken var å refaktorere slik at parameter pathPrefix kunne fjernes ved å skrive /api foran path.replace... Men siden endringslogg ikke skal ha med /api kom jeg ikke frem til en bedre måte å løse dette på. Går tilbake til slik det var. Sender også med stoutLogger nå